pwa/app/lib/rankingPoints.ts

```diff
@@ -96,6 +98,15 @@ export function isScoreBreakdown2024(
   );
 }
 
+export function isScoreBreakdown2025(
+  scoreBreakdown: MatchScoreBreakdown,
+): scoreBreakdown is MatchScoreBreakdown2025Alliance {
+  return (
+    (scoreBreakdown as MatchScoreBreakdown2025Alliance).autoBonusAchieved !==
```

Comment:
I would use coral or barge just in case FIRST reuses autoBonusAchieved in a future game